statistics/histogram.go

```diff
@@ -273,8 +273,13 @@ func HistogramEqual(a, b *Histogram, ignoreID bool) bool {
 const (
 	CurStatsVersion = Version2
 	Version0        = 0
-	Version1        = 1
-	Version2        = 2
+	// In Version1
+	// Column stats: CM Sketch is built in TiKV. Histogram is built from samples. TopN is extracted from CM Sketch.
+	//   TopN + CM Sketch represent all data. Histogram also represents all data.
+	Version1 = 1
+	// In Version2
+	// Column stats: CM Sketch is not used. TopN and Histogram are built from samples. TopN + Histogram represent all data.
+	Version2 = 2
```

Comment:
Can we remove `Version2` since we already have `tidb_analyze_version`?

Reply:
The code is a bit complicated now. 
`tidb_analyze_version` controls how the stats are collected. It is possible that the user has set `tidb_analyze_version` to 2 but the already existent stats are still version 1.
We need to decouple the logic of collecting and using the stats. How to use stats is decided by `stats_ver` in `mysql.stats_histograms`. How to collect stats is decided by `tidb_analyze_version`.

Reply:
Copy that.